### PR TITLE
Improve pre/code blocks with width-control and monospaced fonts

### LINKED ISSUE #148: Improving pre/code blocks with width-control and monospaced fonts
@surreally brought up a great point in the discussion in #145:

* right now, we don't have any width-control mechanism for our `<pre>-<code>` combination - it just overflows
* right now, our font stack for `code` is not particularly readable

With that in mind, we should look to rectify both of these issues! 

* for width-control: we can consider either setting a scrollbar (with `overflow-x: scroll` and a fixed width), or soft-wrapping the lines. To whoever implements this ticket, you should research both options and see what works better!
* for our monospace font: we can consider picking something already on the system (there is a common monospace system font stack), or use an externally-loaded font like Fira Mono. for whoever implements this ticket: we should look into how adding an external font impacts our user experience, and then decide accordingly from there.

### PR DESCRIPTION
**Pull request by EinarBalan**
### Changes

- managed to get the soft wrapping working in pre/code examples, thanks for the helpful links Matt! 
- added Source Code Pro for code examples 
- added myself to the team page

Closes #148

### CODE REVIEW

**Comment by malsf21** [comment]
I'll review this later tonight! One quick thing: you should add something that says "closes #148" or "fixes #148", it'll link the PR and the issue!

File: _team/ebalan.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+firstname: "Einar"
+lastname: "Balan"
+group: "member"
+graduating_year: 2024
+img: ebalan.png
+title: "Python Video Team & Developer"
```
Comment (malsf21) [inline _team/ebalan.md:7]: For secondaries, we have a `secondary` tag; so, it'd be:

```yaml
title: Python Video Team
secondary: Developer
```

File: _team/ebalan.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+github: EinarBalan
+links:
+  - name: "LinkedIn"
+    href: "linkedin.com/in/einarbaan/"
+  - name: "Website"
+    href: "einarbalan.com"
```
Comment (malsf21) [inline _team/ebalan.md:14]: You need to add the `https://` for your links, or it won't work :/

**Review by malsf21 — CHANGES_REQUESTED** [review]
Hey Einar,

Thanks for taking a shot at this PR! Overall, things are looking good, but I've left a bit of feedback on things to do.

1. Your `Source Code Pro` font replacement looks good for `.highlight > pre`! I would also apply it to `.code` as well; if you notice, some blog posts (like Jamie's) have inline code as well, and that should be consistent font-wise!
2. Left a comment in the review on the `head`!
3. Left some comments on the team page

Things are looking good though, especially for your first PR! Excited to merge this soon 😄

**Review by malsf21 — APPROVED** [review]
Thanks for the quick changes, everything looks good to me! Adding the variable for `$font-code` was a smart move, and good job on catching the link bug for Leo's card as well.

I'll let you do the honours of merging, just make sure to use the squash merge option this time!

**Comment by EinarBalan** [comment]
Awesome, just merged. Excited to take on future tickets, this was super fun!

**Pull request merged**